spec/features/short_url_manager_responds_to_furl_requests_spec.rb

```diff
@@ -50,4 +58,16 @@
     expect(mail.subject).to include 'Short URL request denied'
     expect(mail).to have_body_content "Beards are soo last season."
   end
+
+  scenario "Short URL manager edits a short URL" do
+    visit list_short_urls_path
+
+    click_on "Ministry of Hair"
+    click_on "Edit"
+
+    fill_in "Target URL", with: "/government/organisations/minstry-of-long-hair"
+    click_on "Update"
+
+    expect(page).to have_content("Your edit was successful.")
+  end
```

Comment:
It would be good to include an `assert_publishing_api_put_item` in this test to assert that the updated redirect gets sent to publishing-api.